modules/social_features/social_event/asyncapi.yml

```diff
@@ -0,0 +1,139 @@
+channels:
+  eventCreate:
+    address: com.getopensocial.cms.event.create
+    messages:
+      eventCreate:
+        payload:
+          allOf:
+            - $ref: '#/components/schemas/cloudEventsSchema'
+            - type: object
+              properties:
+                data:
+                  type: object
+                  properties:
+                    id:
+                      type: string
+                      description: The UUID of the event.
+                    created:
+                      type: string
+                      format: date-time
+                      description: The creation time of the event.
+                    updated:
+                      type: string
+                      format: date-time
+                      description: The last updated time of the event.
+                    status:
+                      type: string
+                      description: The status of the event.
+                      enum:
+                        - published
+                        - unpublished
+                    label:
+                      type: string
+                      description: The label of the event.
+                    visibility:
+                      type: string
+                      description: The visibility of the event content.
```

Comment:
Shouldn't this be an `object` too?

Either it's **Public**:
```
"visibility": {
  "type": "public"
}
```

**Community**:
```
"visibility": {
  "type": "community"
}
```

**Groups**:
```
"visibility": {
  "type": "groups"
  "groups": ["UUIDv4", etc.]
}
```

**Roles**:
```
"visibility": {
  "type": "roles"
  "roles": ["string", etc.]
}
```

Reply:
I see we currently only display the visibility value here: https://github.com/goalgorilla/open_social/commit/686d9a1e9408ff451178dc647ea4803a5d81f816#diff-98f8f30821c959c87cf6d1df16922c1f498b440f60ed5960f38a3d06c7065961R72.

@ribel, @tregismoreira, based on the initial payload design this should be an object though. Perhaps we can pick it up and address it outside of the scope of this one?
Sorry, I missed this before 😞

Reply:
We agreed to add a followup ticket for that.